src/network/networkd-network-gperf.gperf

```diff
@@ -104,6 +104,7 @@ Network.Xfrm,                                config_parse_stacked_netdev,
 Network.DHCP,                                config_parse_dhcp,                                        0,                             offsetof(Network, dhcp)
 Network.DHCPServer,                          config_parse_bool,                                        0,                             offsetof(Network, dhcp_server)
 Network.LinkLocalAddressing,                 config_parse_link_local_address_family,                   0,                             offsetof(Network, link_local)
+Network.IPv4LLStartAddress,                  config_parse_link_local_start_address,                    0,                             0
```

Comment:
Please specify the offset: `offsetof(Network, ipv4ll_start_address)`

Also, I'd like to move this around `Network.IPv4LLRoute`.